src/main/java/org/codehaus/plexus/classworlds/realm/ClassRealm.java

```diff
@@ -272,6 +272,30 @@ public void addURL( URL url )
         }
     }
 
+    // java11
+    protected Class<?> findClass(String moduleName, String name)
```

Comment:
ah, ok, now I see the same signature as in JDK 11. 
:+1:
The build checks should fail on the code style (put whitespaces around variables and values).

Reply:
Right but the project is not validated (there are ~100 checkstyle errors), added anyway

Reply:
If we are talking about checkstyle errors, then here you should fix yours, e.g. adding white spaces around method parameters.